free-1.md

```diff
@@ -57,12 +95,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think time would be split more towards systems at the beginning, when making an
+MVP. But once we progress past single command=single input, I have been extremely
+conflicted about how to deal with any kind of global state or conditionals, which
```

Comment:
The way I sort of imagine it is that you need to create different "context" to describe different aspects of the game (i.e. movement vs battle vs menu). This seems like it should make describing the game in your language more declarative, and I think you could make this easy to do in your language. There would also probably then be a global context, which handles overall gameplay that doesn't really fit into any of the other categories (i.e. detecting "button" presses, and then passing that information onto contexts that interpret those button presses).